src/main/java/io/github/thebusybiscuit/slimefun4/core/SlimefunRegistry.java

```diff
@@ -72,6 +73,7 @@ public final class SlimefunRegistry {
     private boolean disableLearningAnimation;
     private boolean logDuplicateBlockEntries;
     private boolean talismanActionBarMessages;
+    private SlimefunGuideUnlockMode guideUnlockMode;
```

Comment:
I think a setter method for this would be nice

Reply:
Any demand for setter method? The original goal is implement a research scoped custom unlock provider for addon dev and a global bulit-in unlock provider.